### ISSUE TITLE
Return 403/404 instead of redirecting to https://github.com/minio/console

### ISSUE THREAD

**Issue by bikeshedder** [minio/minio#12223]
## Expected Behavior

When accessing an object that does not exist I would expect a 403/404 HTTP response.

## Current Behavior

When a browser user agent is detected a redirect to https://github.com/minio/console is issued.

When using CURL the correct headers are returned.

## Possible Solution

Do not redirect to https://github.com/minio/console but return a 403/404 response. If you want to be verbose about the console being available put a link to it on the error page.

## Steps to Reproduce (for bugs)

Access any object in a bucket that doesn't exist.

e.g. http://127.0.0.1:9000/this-bucket-does-not-exist/this-object-neither

## Context

I was using minio for local development and testing out S3 policies. I got redirected to the console GitHub page over and over again having to reenter the original URL.

When loading images from minio which don't exist the browser gets redirect to github.com which could even be rated as a violation of GDPR since browsers follow redirects when loading images (and other resources).

## Regression

I think this is a regression. Prior the deprecation of the browser no redirect happened.

## Your Environment

I'm using my own minio quickstart project https://github.com/bikeshedder/minio-quickstart and have tried `minio/minio:latest`, `minio/minio:edge` and `minio/minio:RELEASE.2021-04-27T23-46-03Z.release.0033eb96`.

**Comment by harshavardhana** [comment]
minio/minio:edge - this is a daily master build and this redirection has been long reverted. Try in incognito your browser may cache it.

minio/minio:RELEASE.2021-04-27T23-46-03Z.release.0033eb96 this is an internal release we haven't made any public release on this yet so not sure why you are using this.

 minio/minio:latest - this points to previous release that does not have this code yet. Again a possible reason browser may cache it.

**Comment by bikeshedder** [comment]
I tried different URLs every time, but maybe I've been messing that up. I just tried it today and that solved it. With `MINIO_BROWSER=off` and `minio/minio:latest` it now shows me a proper 403/404 error page. I only get redirected when accessing `/`. Which isn't too bad considering that this URL will rarely every be publicly available. For production minio is going to be hidden behind an nginx or haproxy anyways.

I tried the latest `RELEASE.*` as I wanted to use a specific version rather than relying on the `latest` tag. How does one differentiate between official releases and snapshots then? `minio/console` has tags with version numbers but `minio/minio` just has the `latest`, `edge` and `RELEASE.*` tags.
